web/src/components/dashboard/AnytimeActionDropdown.test.tsx

```diff
@@ -323,5 +323,79 @@ describe("<AnytimeActionDropdown />", () => {
 
       expect(screen.queryByText("license - hvac-reinstatement")).not.toBeInTheDocument();
     });
+
+    it("adds vacant property anytime action for vacant property owners", () => {
```

Comment:
Adding tests for all the filtering on anytime actions from non-essential questions